packages/pluggableWidgets/datagrid-date-filter-web/src/DatagridDateFilter.editorConfig.ts

```diff
@@ -35,32 +35,36 @@ export const getPreview = (values: DatagridDateFilterPreviewProps): StructurePre
                 columnSize: "grow",
                 backgroundColor: "#FFFFFF",
                 children: [
-                    {
-                        type: "Container",
-                        padding: 4,
-                        grow: 0,
-                        children: [
-                            {
-                                type: "Image",
-                                document: getSvgContent(values.defaultFilter)
-                            } as ImageProps
-                        ]
-                    } as ContainerProps,
-                    {
-                        type: "Container",
-                        borders: true,
-                        borderWidth: 0.5,
-                        grow: 0
-                    } as ContainerProps,
+                    ...(values.adjustable
+                        ? [
+                              {
+                                  type: "Container",
+                                  padding: 2,
+                                  grow: 0,
+                                  children: [
+                                      {
+                                          type: "Image",
+                                          document: getSvgContent(values.defaultFilter)
+                                      } as ImageProps
+                                  ]
+                              } as ContainerProps,
+                              {
+                                  type: "Container",
+                                  borders: true,
+                                  borderWidth: 0.5,
+                                  grow: 0
+                              } as ContainerProps
+                          ]
+                        : []),
                     {
                         type: "Container",
                         padding: 8,
                         children: [
                             {
                                 type: "Text",
-                                fontColor: "#BBBBBB",
+                                fontColor: values.placeholder ? "#BBBBBB" : "#FFF",
                                 italic: true,
-                                content: values.placeholder ?? ""
+                                content: values.placeholder.length > 0 ? values.placeholder : "Sample"
```

Comment:
This means that if the user explicitly chooses to fill in an empty placeholder, we disregard that? Is that according to specs? Wouldn't it be better to align it with the above conditional and make it
```suggestion
                                content: values.placeholder ? values.placeholder : "Sample"
```

Reply:
The code above doesnt work in Structure preview, if render the placeholder because we receive empty space (nbsp), thats why it requires .length

Reply:
Fixed